.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -7,6 +7,10 @@ assignees: ''
 
 ---
 
+<!---
+Making your own contribution is easy, encouraged and greatly appreciated! We will put effort into reviewing and merging your PR quickly. For more info, please refer to the contribution guidelines: https://github.com/Adyen/adyen-magento2/blob/develop/CONTRIBUTING.md
+-->
```

Comment:
I just realized that these messages are in comment tags, these wouldn't be visible this way @tnaber 
```suggestion
Making your own contribution is easy, encouraged and greatly appreciated! We will put effort into reviewing and merging your PR quickly. For more info, please refer to the contribution guidelines: https://github.com/Adyen/adyen-magento2/blob/develop/CONTRIBUTING.md
```